### PR TITLE
[TypeScript][Skill] Deployment Scripts

### PR DESCRIPTION
**Pull request by matiasroldan6**
### Purpose
Update all the deployment scripts from the TS Skill bot to v1.0

### CODE REVIEW

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 [hunk 1]
```diff
@@ -142,72 +224,94 @@ Write-Host "Done." -ForegroundColor Green
...
+
+    # To explain the syntax here:
+    # - 'az deployment group validate' is being executed with supplied parameters prefixed with '--'
+    # - 2>&1 merges the stderr output stream into stdout, ensuring all output from the executed command comes through stdout
+    # - stdout is piped into Tee-Object to write the contents of stdout to our log file, and capture the piped contents in a variable, $validation
+    # - The stream is finally piped on into Out-Null so that it does not get rendered to the host
+    #
+    az deployment group validate `
+        --resource-group $resourcegroup `
+        --template-file "$(Join-Path $PSScriptRoot '..' 'resources' 'template.json')" `
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:237]:
```suggestion
        --template-file "$(Join-Path $PSScriptRoot '..' 'Resources' 'template.json')" `
```

Just to follow the parity. It's ok to be `Resources`

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 [hunk 2]
```diff
@@ -142,72 +224,94 @@ Write-Host "Done." -ForegroundColor Green
...
+    }
+
+    Write-Host "Done." -ForegroundColor Green
+
+    Write-Host "> Deploying Azure services (this could take a while)..." -ForegroundColor Yellow -NoNewline
+
+    az deployment group create `
+        --name $timestamp `
+        --resource-group $resourceGroup `
+        --template-file "$(Join-Path $PSScriptRoot '..' 'resources' 'template.json')" `
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:265]:
```suggestion
        --template-file "$(Join-Path $PSScriptRoot '..' 'Resources' 'template.json')" `
```
Same

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 [hunk 3]
```diff
@@ -142,72 +224,94 @@ Write-Host "Done." -ForegroundColor Green
...
+
+    # To explain the syntax here:
+    # - 'az deployment group validate' is being executed with supplied parameters prefixed with '--'
+    # - 2>&1 merges the stderr output stream into stdout, ensuring all output from the executed command comes through stdout
+    # - stdout is piped into Tee-Object to write the contents of stdout to our log file, and capture the piped contents in a variable, $validation
+    # - The stream is finally piped on into Out-Null so that it does not get rendered to the host
+    #
+    az deployment group validate `
+        --resource-group $resourcegroup `
+        --template-file "$(Join-Path $PSScriptRoot '..' 'resources' 'template.json')" `
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:283]:
```suggestion
        --template-file "$(Join-Path $PSScriptRoot '..' 'Resources' 'template.json')" `
```
Same

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 [hunk 4]
```diff
@@ -142,72 +224,94 @@ Write-Host "Done." -ForegroundColor Green
...
+    }
+
+    Write-Host "Done." -ForegroundColor Green
+
+    Write-Host "> Deploying Azure services (this could take a while)..." -ForegroundColor Yellow -NoNewline
+
+    az deployment group create `
+        --name $timestamp `
+        --resource-group $resourceGroup `
+        --template-file "$(Join-Path $PSScriptRoot '..' 'resources' 'template.json')" `
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:310]:
```suggestion
        --template-file "$(Join-Path $PSScriptRoot '..' 'Resources' 'template.json')" `
```
Same

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 (hunk 1 above)
Comment (matiasroldan6) [reply templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:237]: Are you sure we won't have any case-sensitivity problem, maybe running under a different system? I'll change it, anyway.

Comment (Batta32) [reply templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:237]: I'm not sure the scenario of executing under different OS. You can review this if you want.

Meanwhile, we can use the C# pattern!

File: templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1 [hunk 5]
```diff
@@ -215,7 +214,7 @@ if (-not $armLuisAuthoringRegion) {
 }
 
 # Get timestamp
-$timestamp = Get-Date -Format MMddyyyyHHmmss
+$timestamp = Get-Date -f MMddyyyyHHmmss
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/deployment/scripts/deploy.ps1:217]: I don't think that this is ok, following [C#](https://github.com/southworks/botframework-solutions/blob/master/samples/csharp/skill/SkillSample/Deployment/Scripts/deploy.ps1#L213), this change was introduced [here](https://github.com/southworks/botframework-solutions/commit/403746704325144dadd9c98bccfef4a6454a6233)

**Pull request merged**